justfile

```diff
@@ -92,6 +95,8 @@ integration:
 unit:
     docker compose run --rm api-unit
 
+ingest_unit:
+    docker compose run --rm ingest-unit
```

Comment:
So there were already ingest unit tests, they were just not part of `just`?

Reply:
Yeah, I just made a `Dockerfile` for the tests and the `just` commands.